PWGHF/HFC/DataModel/CorrelationTables.h

```diff
@@ -470,24 +470,47 @@ DECLARE_SOA_TABLE(DmesonSelection, "AOD", "DINCOLL", // Selection of D meson in
                   aod::hf_selection_dmeson_collision::DmesonSel);
 
 // Note: definition of columns and tables for Electron Hadron correlation pairs
+namespace hf_electron
+{
+DECLARE_SOA_COLUMN(PhiElectron, phiElectron, float); //! Phi of electron
+DECLARE_SOA_COLUMN(EtaElectron, etaElectron, float); //! Eta of electron
+DECLARE_SOA_COLUMN(PtElectron, ptElectron, float);   //! Transverse momentum of electron
+DECLARE_SOA_COLUMN(NLSEl, nLSEl, int);               //! like sign Electron
+DECLARE_SOA_COLUMN(NULSEl, nULSEl, int);             //! unLike sign Electron
```

Comment:
These names are too cryptic and don't follow the naming scheme.
```suggestion
DECLARE_SOA_COLUMN(NElectronsLS, nElectronsLS, int);               //! number of like-sign electrons
DECLARE_SOA_COLUMN(NElectronsUS, nElectronsUS, int);             //! number of unlike-sign electrons
```

Reply:
okay